package.json

```
@@ -40,9 +40,9 @@
     "cytoscape-svg": "0.2.0",
     "lodash": "^4.17.11",
     "ramda": "^0.25.0",
-    "react": "15.4.2",
+    "react": "^16.13.1",
```

I wonder if it's better to fix it to 16.13.0 since that's what's done on `plotly/dash-core-components`. Not sure if it might break dash if react/react-dom are upgraded beyond that.
```suggestion
    "react": "16.13.0",
```